src/it/resources/com/google/checkstyle/test/chapter4formatting/rule413emptyblocks/InputEmptyBlocksAndCatchBlocks.java

```diff
@@ -31,6 +34,44 @@ public int doSideEffect() {
   }
 
   public void emptyMethod() {}
+
+  void foo() throws Exception {
+    int a = 90;
+
+    if (a == 1) {
+    } else {}
+
+    if (a == 1) {
+    } else { } // violation 'Empty blocks should have no spaces.'
+    // false negative above
```

Comment:
nope, it is not false negative.
Message is long and it ends with `Empty blocks may only be represented as '{}' when not part of a multi-block statement (4.1.3)`

can we try:
`// violation "Empty blocks should have no spaces. .* may only be represented as '{}'"`

Reply:
done, in some places I had so trim your suggested message because it was crossing column width limit